src/Button/ZoomToExtentButton/ZoomToExtentButton.tsx

```diff
@@ -93,6 +110,9 @@ class ZoomToExtentButton extends React.Component<ZoomToExtentButtonProps> {
     if (!view) { // no view, no zooming
       return;
     }
+    if (!extent && (!center && !zoom)) {
```

Comment:
should read
```
!extent && (!center || !zoom)
```
as using only zoom or only center should also fail / raise a message

Reply:
or you keep that line and change the behaviour below to zoom only when zoom is given and center only when center is given.